### PR TITLE
Background media controls

### PR DESCRIPTION
**Pull request by akinwale**

### CODE REVIEW

File: src/main/java/io/lbry/browser/reactmodules/BackgroundMediaModule.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import io.lbry.browser.R;
+
+public class BackgroundMediaModule extends ReactContextBaseJavaModule {
+
+    private static final int NOTIFICATION_ID = 900;
+
+    private static final String NOTIFICATION_CHANNEL_ID = "io.lbry.browser.MEDIA_PLAYER_NOTIFICATION_CHANNEL";
+
+    public static final String ACTION_PLAY = "io.lbry.browser.ACTION_MEDIA_PLAY";
+
```
Comment (skhameneh) [inline src/main/java/io/lbry/browser/reactmodules/BackgroundMediaModule.java]: Some of these line breaks aren't needed here

File: src/main/java/io/lbry/browser/reactmodules/BackgroundMediaModule.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    }
+
+    @ReactMethod
+    public void showPlaybackNotification(String title, String publisher, String uri, boolean paused) {
+        if (!channelCreated && Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            NotificationManager notificationManager =
+                (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+            NotificationChannel channel = new NotificationChannel(
+                NOTIFICATION_CHANNEL_ID, "LBRY Media", NotificationManager.IMPORTANCE_LOW);
+            channel.setDescription("LBRY media player");
```
Comment (skhameneh) [inline src/main/java/io/lbry/browser/reactmodules/BackgroundMediaModule.java]: Does this render within the notification?
Maybe we should adjust this text a bit.

Comment (skhameneh) [reply src/main/java/io/lbry/browser/reactmodules/BackgroundMediaModule.java]: I'll need to play with this a bit on my device

Comment (akinwale) [reply src/main/java/io/lbry/browser/reactmodules/BackgroundMediaModule.java]: No, it doesn't. Notification channels are usually configured in notification settings on Android 8.0 and higher. Basically, you can turn channels on or off which lets you determine the notifications related to channels that you want to see. https://developer.android.com/training/notify-user/channels

File: src/main/java/io/lbry/browser/reactmodules/BackgroundMediaModule.java (hunk 1 above)
Comment (akinwale) [reply src/main/java/io/lbry/browser/reactmodules/BackgroundMediaModule.java]: I usually format this way to improve readability. Also helps when adding Javadocs.

**Pull request merged**